lfs/lfs.go

```diff
@@ -16,10 +16,6 @@ import (
 	"github.com/rubyist/tracerx"
 )
 
-const (
-	Version = "1.5.0"
```

Comment:
I can do that. It also solves the issue of `Version` being a `var` in `config/version.go`. IMO it should be a `const` like it is here.